packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -436,16 +437,17 @@ async function handler(
     luckyUsers,
     routedTeamMemberIds,
     reroutingFormResponses,
-    routingFormResponseId,
+    queuedFormResponse,
     _isDryRun: isDryRun = false,
     _shouldServeCache,
     ...reqBody
   } = bookingData;
-
   let troubleshooterData = buildTroubleshooterData({
     eventType,
   });
 
+  let routingFormResponseId = bookingData.routingFormResponseId;
```

Comment:
The `routingFormResponseId` might change depending if the form response is queued or not. Once we write the form response to the actual table then we need to update the `routingFormResponseId` to point to the record in the response table